srcpkgs/herbe/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'herbe'
+pkgname=herbe
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+makedepends="libXft-devel freetype-devel"
+short_desc="Daemon-less notifications without D-Bus. Minimal and lightweight"
+maintainer="tuxliban <tenshalito@gmail.com>"
+license="MIT"
+homepage="https://github.com/dudik/herbe"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=78e454159050c86e030fb5a6cf997ac914345210cdf5a4ca4d7600c5296b7f76
+patch_args="-Np1"
+
+do_build() {
+	export CFLAGS=${CFLAGS}
```

Comment:
this 
1. doesn't build anything and renders the [package empty (except license)](https://github.com/void-linux/void-packages/pull/27195/checks?check_run_id=1563142458#step:13:9)
2. should not be necessary as CFLAGS is already in env

Reply:
Previously I tried to build the binary directly with the help of the gnu-makefile build script, but despite having specified the makedepends in the middle of the process it gives me an error for missing a header.
The only way I was able to build the binary was as the template currently is.

```
-> $ ./xbps-src -f pkg herbe
=> xbps-src: updating repositories for host (x86_64-musl)...
[*] Updating repository `https://alpha.us.repo.voidlinux.org/current/musl/x86_64-musl-repodata' ...
x86_64-musl-repodata: 1688KB [avg rate: 86KB/s]
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/x86_64-musl-repodata' ...
x86_64-musl-repodata: 1688KB [avg rate: 187KB/s]
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/nonfree/x86_64-musl-repodata' ..
=> xbps-src: updating software in / masterdir...
=> xbps-src: cleaning up / masterdir...
=> herbe-1.0.0_1: removing autodeps, please wait...
=> herbe-1.0.0_1: building [gnu-makefile] for x86_64-musl...
   [target] libXft-devel-2.3.3_1: found (https://alpha.de.repo.voidlinux.org/current/musl)
   [target] freetype-devel-2.10.4_1: found (https://alpha.de.repo.voidlinux.org/current/musl)
=> herbe-1.0.0_1: installing target dependencies: libXft-devel-2.3.3_1 freetype-devel-2.10.4_1 ...
=> herbe-1.0.0_1: running do-fetch hook: 00-distfiles ...
=> herbe-1.0.0_1: running do-extract hook: 00-distfiles ...
=> herbe-1.0.0_1: extracting distfile(s), please wait...
=> herbe-1.0.0_1: running do-patch hook: 00-patches ...
=> herbe-1.0.0_1: patching: 11.diff.
=> herbe-1.0.0_1: patching: Xresources.diff.
Reversed (or previously applied) patch detected!  Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file config.def.h.rej
Reversed (or previously applied) patch detected!  Skipping patch.
4 out of 4 hunks ignored -- saving rejects to file herbe.c.rej
=> ERROR: herbe-1.0.0_1: do-patch_00-patches: 'patch -sl ${_args} -i ${_patch} 2> /dev/null' exited with 1
=> ERROR:   in _process_patch() at common/hooks/do-patch/00-patches.sh:34
=> ERROR:   in hook() at common/hooks/do-patch/00-patches.sh:51
=> ERROR:   in run_func() at common/xbps-src/shutils/common.sh:21
=> ERROR:   in run_pkg_hooks() at common/xbps-src/shutils/common.sh:245
=> ERROR:   in run_step() at common/xbps-src/shutils/common.sh:71
=> ERROR:   in main() at common/xbps-src/libexec/xbps-src-dopatch.sh:33
```

Reply:
as the template is now, the package just ships one file (the license)